share/completions/mkvextract.fish

```diff
@@ -0,0 +1,47 @@
+# Sample output of 'mkvmerge -i file.mkv'
+# 
+# File 'file.mkv': container: Matroska
+# Track ID 0: video (MPEG-4p10/AVC/h.264)
+# Track ID 1: audio (AAC)
+# Track ID 2: subtitles (SubStationAlpha)
+# Attachment ID 1: type 'application/x-truetype-font', size 53532 bytes, file name 'some_font.ttf'
+# Chapters: 7 entires
+
+function __fish_print_mkvextract_attachments
+    mkvmerge -i $argv | string match 'Attachment ID*' | string replace -r '.*?(\d+).*? type \'(.*?)\'.*?file name \'(.*?)\'' '$1:\t$3 ($2)'
```

Comment:
Yes, how it works is you actually specify 
`mkvextract tracks file.mkv 2:audio.aac 3;subtitles.ass`
The name of the output file is up to the user, the completion only proposes available streams.